Exploration Queries/SecurityEvent/ParentProcessesOnHost.txt

```diff
@@ -29,7 +29,7 @@ and ParentProcessName !contains ":\\Windows\\CCM\\CcmExec.exe"
 | project min_TimeGenerated, max_TimeGenerated, EventID, Account, AccountType, Computer, ParentProcessName, NewProcessName, TargetAccount, CommandLine
 | top 10 by min_TimeGenerated asc
 // rename the relevant columns to the format {V3EntityType_FieldName}
-| project-rename Host_NetBiosName=Computer, Account_DisplayName=Account, Process_CommandLine=CommandLine, Process_ImageFile_FullPath=NewProcessName, Process_ParentProcess_ImageFile_FullPath=ParentProcessName
+| project-rename Process_Host_NetBiosName=Computer, Process_Account_DisplayName=Account, Process_CommandLine=CommandLine, Process_ImageFile_FullPath=NewProcessName, Process_ParentProcess_ImageFile_FullPath=ParentProcessName
```

Comment:
The convention is to **always have the name of the output entity type as a prefix to the nested fields**.
In the example above, the query returns a Process, and the Process has a Host field within it. So the column representing the HostName of that field would be written as: **Process_Host_HostName**.

This is to allow for returning several entities as an output without having ambiguous naming.

If the same query is to return an Account, and that account also has a Host which has a HostName, then the column representing that would be written as: **Account_Host_HostName**